lib/active_force/active_query.rb

```diff
@@ -33,12 +33,6 @@ def limit limit
       limit == 1 ? to_a.first : self
     end
 
-    def where args=nil, *rest
```

Comment:
Yea definitely. I needed a quick and dirty solution to get something out the door and figured it might as well be the start of a discussion. I'll think about it for real now.